docs/src/pages/components/data-grid/data/data.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Data Grid - Data
+components: DataGrid
+---
+
+# Data Grid - Data
+
+<p class="description">A fast and extendable data table and data grid for React. It's a feature-rich compoent available in MIT or Enterprise versions.</p>
+
```

Comment:
might be worth adding a section on row and cell rendering since that's a pretty common use case